src/Alert.php

```diff
@@ -139,48 +301,85 @@ private function renderBodyEnd(): string
      *
      * @return string the rendering result
      */
-    private function renderCloseButton(): ?string
+    public function renderCloseButton(bool $outside = true): ?string
     {
-        if ($this->closeButtonEnabled === false) {
+        if ($this->closeButton === null) {
             return null;
         }
 
-        $tag = ArrayHelper::remove($this->closeButton, 'tag', 'button');
-        $label = ArrayHelper::remove($this->closeButton, 'label', '');
+        $options = array_merge(
+            $this->closeButton,
+            [
+                'aria-label' => 'Close',
+                'data-bs-dismiss' => 'alert',
+            ],
+        );
+        $tag = ArrayHelper::remove($options, 'tag', 'button');
```

Comment:
Why not add a `tag()` method, and remove it from the  `options`, it is much better.